Scripts/Python/psnlYeeshaPageChanges.py

```diff
@@ -63,19 +63,10 @@
 respDisable = ptAttribResponder(6, "Disabled resp (if necessary)")
 
 #globals
-TotalPossibleYeeshaPages = 25
+TotalPossibleYeeshaPages = 26
```

Comment:
Might be more future-proof to:
```suggestion
import xLinkingBookDefs
TotalPossibleYeeshaPages = len(xLinkingBookDefs.xYeeshaPages)
```

So this value doesn't have to be updated when new pages are added.